eth/chains/base.py

```diff
@@ -119,20 +118,19 @@
         take,
     )
 
-if TYPE_CHECKING:
-    from eth.vm.base import (     # noqa: F401
-        BaseVM,
-    )
+from eth.vm.base import (     # noqa: F401
```

Comment:
I think the noqa can be dropped now

Reply:
Do I remove all? `#noqa: F401`

Reply:
Ah! The noqa is for linting errors.

Reply:
`BaseVM` is being used in line 132 now. That's why I don't expect the "unused variable" lint error to show up anymore. When you run `flake8` locally without the `noqa`, you're still seeing it?

Reply:
Removing it, I have to move it up to fix the lint error